CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 ## [Unreleased]
 
+## [0.0.3] - 2018-02-24
+
+### Added
+* README: add section `"Who Uses Airtap?"` (@feross)
+* Test aggregation of available browsers for `--list-browsers` (@ralphtheninja)
+
+### Changed
+* Rename `lib/zuul.js` to `lib/airtap.js` (@yeskunall)
+* Replace `char-split` with `split2` (@jeffreyshen19)
+* Replace `colors` with `chalk` (@yeskunall)
+
+### Removed
+* Remove vim styling and `.editorconfig` (@moondef)
+* Remove all frameworks but `tape` (@ralphtheninja)
+* Remove `.ui/--ui` option (@ralphtheninja)
+* Remove `mocha` dependency (@ralphtheninja)
+* Remove references to `emberjs` in docs and change example to use `tape` (@ralphtheninja)
+
+### Fixed
+* Fix broken `--list-available-browsers` and change to `-l/--list-browsers` (@ralphtheninja)
```

Comment:
Can you split this in two? One entry under Changed ("Rename `--list-available-browsers` to `-l/--list-browsers`") and one under Fixed.